src/op/persistent_save.lua

```diff
@@ -0,0 +1,13 @@
+local e = _G.PolarisNav
+local p = e.plugin
+local stringify = e.stringify
+
+return function (key, value)
+    if typeof(value) == "string" then
+        p:SetSetting(key, "#" .. value)
+    elseif typeof(value) == nil then
+        p:SetSetting(key, value)
+    else
+        p:SetSetting(key, "!" .. stringify(value))
```

Comment:
Ooo, this is likely converting numbers and booleans as well. We should be able to save those and booleans directly without issue.

Reply:
Made that work.